src/css/style.css

```diff
@@ -471,6 +478,14 @@ button {
     content: "\e60c";
 }
 
+.gpii-fd-navIcon-outer {
+    width: 30rem; /* Space for 5 icons */
+    white-space: nowrap;
+    overflow: hidden;
+    padding-top: 1.5rem; /* gap between main body and icons*/
+    margin-left: 0;
```

Comment:
The nav icon bar is not centered. Adjusting, or removing, "margin-left" seems helpful.